src/test/unit/test_lumigo_utils.py

```diff
@@ -247,12 +248,50 @@ def func():
             f'{{"{SKIP_SCRUBBING_KEYS[0]}": {{"password": 1}}}}',
         ),
         ([{"password": 1}, {"a": "b"}], '[{"password": "****"}, {"a": "b"}]'),  # list of dicts
+        (  # Dict of long list
+            {"a": [{"key": "value", "password": "value", "b": "c"}]},
+            '{"a": [{"key": "****", "password": "****", "b": "c"}]}',
+        ),
+        (  # Multiple nested lists
+            {"a": [[[{"c": [{"key": "v"}]}], [{"c": [{"key": "v"}]}]]]},
+            '{"a": [[[{"c": [{"key": "****"}]}], [{"c": [{"key": "****"}]}]]]}',
+        ),
     ],
 )
 def test_lumigo_dumps(value, output):
     assert lumigo_dumps(value, max_size=100) == output
 
 
+@pytest.mark.parametrize(
+    ("value", "omit_skip_path", "output"),
+    [
+        ({"a": "b", "Key": "v"}, ["Key"], '{"a": "b", "Key": "v"}'),  # Not nested
+        (  # Nested with list
+            {"R": [{"o": {"key": "value"}}]},
+            ["R", "o", "key"],
+            '{"R": [{"o": {"key": "value"}}]}',
+        ),
+        (  # Doesnt affect other paths
+            {"a": {"key": "v"}, "b": {"key": "v"}},
+            ["a", "key"],
+            '{"a": {"key": "v"}, "b": {"key": "****"}}',
+        ),
+        (  # Nested items not affected
+            {"key": {"password": "v"}},
+            ["key"],
+            '{"key": {"password": "****"}}',
+        ),
+        (  # Skipping nested item should skip also parent item
+            {"key": {"password": "v"}},
+            ["key", "password"],
+            '{"key": {"password": "v"}}',
+        ),
```

Comment:
There is another test that you should add:
`{"a": {"key": "c"}}, ["key"], {"a": {"key": "c"}}`
Which is: you need that the whole path will match, and not only a suffix of it.